sdk/src/main/java/velocitas/sdk/VehicleApp.kt

```diff
@@ -0,0 +1,47 @@
+package velocitas.sdk
+
+import velocitas.sdk.middleware.Middleware
+
+/**
+ * Base class for all vehicle apps which manages an app's lifecycle.
+ */
+abstract class VehicleApp
+{
+
+    /**
+     * Runs the Vehicle App.
+     */
+    fun run() {
+        Logger.info("Running App...")
+        Middleware.getInstance().start()
+        Middleware.getInstance().waitUntilReady()
+
+        onStart()
+
+        // TODO: Fix busy waiting
+        while (true) {
+            Thread.sleep(1)
+        }
```

Comment:
Took it over from here: https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/sdk/include/sdk/VehicleApp.h
However I don't know what purpose it aims for